Sources/Recycling/ListViews/AdManagement/Cells/UserAdManagementStatisticsCell.swift

```diff
@@ -33,17 +46,22 @@ public class UserAdManagementStatisticsCell: UITableViewCell {
             stackView.removeArrangedSubview(oldSubview)
             oldSubview.removeFromSuperview()
         }
-        let lastIndex = itemModels.count-1
-
-        for (index, model) in itemModels.enumerated() {
-            let itemView = StatisticsItemView(model: model)
-            itemView.shouldShowLeftSeparator = index > 0 && index < lastIndex
-            itemView.shouldShowRightSeparator = index > 0 && index < lastIndex
-            // Slight overengineering, as there are currently no plans to receive any other number than three items
-            if lastIndex == 1 { itemView.shouldShowLeftSeparator = index == lastIndex }
-            if lastIndex > 2 { itemView.shouldShowRightSeparator = index == lastIndex-1 }
-            stackView.addArrangedSubview(itemView)
-            itemView.setupConstraints()
+
+        if itemModels.isEmpty {
+            stackView.addArrangedSubview(emptyView)
+        } else {
+            let lastIndex = itemModels.count-1
+
+            for (index, model) in itemModels.enumerated() {
+                let itemView = StatisticsItemView(model: model)
+                itemView.shouldShowLeftSeparator = index > 0 && index < lastIndex
+                itemView.shouldShowRightSeparator = index > 0 && index < lastIndex
+                // Slight overengineering, as there are currently no plans to receive any other number than three items
+                if lastIndex == 1 { itemView.shouldShowLeftSeparator = index == lastIndex }
+                if lastIndex > 2 { itemView.shouldShowRightSeparator = index == lastIndex-1 }
+                stackView.addArrangedSubview(itemView)
+                itemView.setupConstraints()
```

Comment:
Could you move this call to one of the lifecycle methods on the view itself? I.e. `viewDidLoad()`?

Reply:
I see now that you didn't add this loc and that it's added to a stackView. Don't mind me :)